pages/repositories.html

```diff
@@ -10,18 +10,18 @@
 	 <meta name="author" content="Junior Developer Group" />
 	 <meta
 	   name="description"
-	   content="Manage projects with personalized dashboard"
+	   content="Junior Developer Group | Dashboard page containing all current and previous projects"
 	 />
  
 	 <!-- SEO SETTINGS FOR WHEN WEBSITE URL IS SHARED -->
 	 <meta
 	   name="og:title"
 	   property="og:title"
-	   content="JDG Dashboard"
```

Comment:
I've updated 'JDG' to group's full name as 'JDG' can be many things :P we want to be using our full group name for pages like this

Reply:
That's true~ I was just keeping it same as the page's title (the one showing on tab) 😂